pkgs/applications/office/spice-up/0001-Remove-environment-dependant-CMake-configuration.patch

```diff
@@ -0,0 +1,46 @@
+From d6bfe4f4584a575209e790cc31756411925c73f6 Mon Sep 17 00:00:00 2001
+From: Sam Parkinson <sam@sam.today>
+Date: Sat, 3 Feb 2018 13:16:16 +1100
+Subject: [PATCH] Remove environment dependant CMake configuration
+
+This is broken by Nix's CMake patching scripts
```

Comment:
Do you have a reference for this? In the setup hook, I only see a borked [path replacer](https://github.com/NixOS/nixpkgs/blob/27a4a5511dc2cf9756943458d7a752617e4ad6e3/pkgs/development/tools/build-managers/cmake/setup-hook.sh#L10).

Reply:
yeah sorry that was bullshit